packages/prettier-config/README.md

```diff
@@ -8,6 +8,30 @@ First, install the package, and make sure you've upgraded Prettier to >= 1.17.0:
 yarn add --dev @stellar/prettier-config prettier
 ```
 
+You almost always want to run prettier on pre-commit, so no one ever has to
+think about it. So set up `husky` and `lint-staged` if you haven't already:
+
+```
+yarn add --dev husky lint-staged
+```
+
+And add the config, as needed, to the top level of your `package.json`:
+
+```
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
```

Comment:
I'm slightly averse to `lint-staged`, it overwrites contents and `git add`s the entire file. I like to stage and commit lines sometimes, which isn't possible there.

On stellar.org I set it up with `pretty-quick --staged`, which verifies that what's staged matches prettier output and returns a nonzero exit code if not.